Build/Module.xml

```diff
@@ -4,7 +4,7 @@
   <DefaultAction>Generate</DefaultAction>
   <DefaultLinuxPlatforms>Android,Angle,Linux,WindowsGL</DefaultLinuxPlatforms>
   <DefaultMacOSPlatforms>Angle,MacOS,iOS,WindowsGL,Android,tvOS,Linux</DefaultMacOSPlatforms>
-  <DefaultWindowsPlatforms>Android,Angle,Linux,Windows,WindowsGL,WindowsUniversal,iOS,tvOS</DefaultWindowsPlatforms>
```

Comment:
You have broken Pipeline Tool generation by removing this.

Reply:
We can keep it until we migrate tools, will undo.